lib/couchbase-orm/persistence.rb

```diff
@@ -230,7 +235,7 @@ def _update_record(with_cas: false, **options)
 
                     _id = @__metadata__.key
                     options[:cas] = @__metadata__.cas if with_cas
-
+                    CouchbaseOrm.logger.debug "Data - Replace #{_id} #{@__attributes__.to_s.truncate(200)}"
```

Comment:
attention le to_s peut couter chère en performance et il sera appeler tout le temps

Reply:
I can replace with:
```
CouchbaseOrm.logger.debug { "Data - Replace #{_id} #{@__attributes__.to_s.truncate(200)}" }
```
have you another suggestion?